src/dev/mod.ts

```diff
@@ -107,41 +107,28 @@ function toImportSpecifier(file: string) {
   return specifier;
 }
 
+/** @todo(iuioiua) Try without `JSON.stringify()` */
+function toPath(file: string) {
+  return JSON.stringify(`${toImportSpecifier(file)}`);
+}
+
 export async function generate(directory: string, manifest: Manifest) {
   const { routes, islands } = manifest;
 
   const output = `// DO NOT EDIT. This file is generated by Fresh.
 // This file SHOULD be checked into source version control.
 // This file is automatically updated during development when running \`dev.ts\`.
 
-${
-    routes.map((file, i) =>
-      `import * as $${i} from "${toImportSpecifier(file)}";`
-    ).join(
-      "\n",
-    )
-  }
-${
-    islands.map((file, i) =>
-      `import * as $$${i} from "${toImportSpecifier(file)}";`
-    )
-      .join("\n")
-  }
-
 const manifest = {
   routes: {
     ${
-    routes.map((file, i) =>
-      `${JSON.stringify(`${toImportSpecifier(file)}`)}: $${i},`
-    )
+    routes.map((file) => `${toPath(file)}: await import(${toPath(file)}),`)
```

Comment:
This will cause the route to be eagerly imported the same way as before, just with an additional promise tick. Did you meant to go with `() => await import(...)` instead?

Reply:
Yeah, that was intentional. I don't see the benefit in them being functions that return the awaited imports.